packages/phrases/src/locales/ko/translation/admin-console/general.ts

```diff
@@ -2,6 +2,7 @@ const general = {
   placeholder: '플레이스홀더',
   skip: '건너뛰기',
   next: '다음',
+  back: 'Back', // UNTRANSLATED
```

Comment:
nit: I think we'd better translate these phrases for cloud version, as this is not just for open source any more.

Reply:
An issue added: LOG-5417